client/store.js

```diff
@@ -244,6 +254,12 @@ store.setupKresus = function(cb) {
         has(world, 'accounts');
         has(world, 'operations');
 
+        let defaultAccountId = store.getDefaultAccountId();
+        if (defaultAccountId && !data.currentAccountId)
+        {
+            data.currentAccountId = defaultAccountId;
```

Comment:
What becomes the assert then ?
I don't really find it as more readable though ?